appium/lib/screen-object/accessors.rb

```diff
@@ -15,529 +15,654 @@
 =end
 
 module ScreenObject
-
   # contains  module level methods that are added into your screen objects.
   # when you include the ScreenObject module.  These methods will be generated as services for screens.
 
   # include Gem 'screen-object' into project Gemfile to add this Gem into project.
   # include require 'screen-object' into environment file. doing this , it will load screen object methods for usage..
 
   module Accessors
-
     # Button class generates all the methods related to different operations that can be performed on the button.
     def button(name, locator)
+      # generates method for clicking button.
```

Comment:
Not sure what exactly got changed in this file. Almost all methods are shown as removed and then shown as added. Can you simply revert this file to what it was to avoid unnecessary review of such a large file?